includes/settings/tools.php

```diff
@@ -128,21 +128,26 @@ public function post_elementor_rollback() {
 
 		$rollback_versions = $this->get_rollback_versions();
 		$version = Utils::get_super_global_value( $_GET, 'version' );
+		$is_cloud = strpos( ELEMENTOR_VERSION, '-cloud' ) !== false;
 
-		if ( empty( $version ) || ! in_array( $version, $rollback_versions, true ) ) {
+		if ( ! $is_cloud && empty( $version ) || ! in_array( $version, $rollback_versions, true ) ) {
```

Comment:
Why needed?

Reply:
It should be the default behavior, if the plugin version is not cloud, just act as usual